### ISSUE TITLE
Measure-attached texts

### ISSUE THREAD

**Issue by lgvr123** [musescore/MuseScore#17198]
### Your idea

Give the possibility for text elements to be optionality attached (or laid-out) relatively to the measure and not only to the element they have been added to.
![layoutHelps-Anchors](https://user-images.githubusercontent.com/50494/231083866-26e9fd0e-d751-40ee-b144-20b9f6c5fb1b.png)

This feature already for some text types (*) but should be available for any type of text, to the discretion of the user.

### Problem to be solved

Except some text-types (*) all the text elements are laid-out relatively to the element they have been added to.
It is therefore complicated to position a text element at the end or the start of the measure, because if the measure size changes for whatever reason, the text looses its relative position to the measure boundaries.

In this example a "3x" has been added to the last element to indicate the number of repetitions.
A simple carriage return would enlarge the measure and _de facto_ move the "3x" relatively to the end of the measure.

![align](https://user-images.githubusercontent.com/50494/231084770-c73cb491-bdbf-4a6d-962d-291cd937a120.png)

----
(*): such as repetition marks ("To Coda", "Coda", ...) that are already attached to the measure, 

### Additional context

See discussion [Copy Part Layout](https://musescore.org/en/node/347363#comment-1184992)

**Comment by GabeS573** [comment]
I would also like to see "center of measure" as an option, which would finally allow automatic positioning for numbers over repeat symbols:
![image](https://user-images.githubusercontent.com/91428485/231309058-62ae8aa5-9094-4c9d-95e2-68f6cf038e07.png)

**Comment by MarcSabatella** [comment]
I agree center is needed, but - numbering of measure repeats is supported directly.  See Format / Style / Measure Repeats.

**Comment by GabeS573** [comment]
> I agree center is needed, but - numbering of measure repeats is supported directly. See Format / Style / Measure Repeats.

Ah, I didn't know that setting existed—but it doesn't look like the font used for the numbers can be changed (unless I'm missing something else). Probably a separate feature request, then.

**Comment by oktophonie** [comment]
Measure repeat font: https://github.com/musescore/MuseScore/issues/13605

**Comment by GabeS573** [comment]
I literally commented in that thread then forgot about it 🤦‍♂️

**Comment by jim-miller** [comment]
I know this has come up in other feature requests, and it sounds like it's on peoples' radar, but I would give a +1 to centering text as mentioned above.

A common example in drum charts that comes up for me during long "play time" sections:

<img width="433" alt="drumnotationtimeoption" src="https://github.com/musescore/MuseScore/assets/229085/a4e4c0f2-91f6-4657-bb48-7126f60e1c11">
